docs/index.md

```diff
@@ -20,11 +50,11 @@ Operators are the way to go. This SDK will make it easy for Java developers to e
 * Safer deployment of applications - only expose cluster to users by Custom Resources
 
 # Roadmap
+* Comprehensive documentation
+* Integrate with operator-sdk to generate project skeleton 
 * Testing of the framework and all samples while running on a real cluster.
-* Generate a project skeleton
 * Generate Java classes from CRD definion (and/or the other way around)
-* Integrate with Quarkus (including native image build)
-* Integrate with OLM (Operator Lifecycle Manager)
+
```

Comment:
I'd leave OLM support, though that's not high priority at the moment.

Reply:
me too :) but we didn't even mention it for a while so I thought it's better to just keep more current things here.